pom.xml

```diff
@@ -15,8 +15,8 @@
 
   <properties>
     <aspectj.version>1.9.7</aspectj.version>
-    <raml-module-builder.version>34.1.0</raml-module-builder.version>
-    <vertx.version>4.3.1</vertx.version>
+    <raml-module-builder.version>35.0.0</raml-module-builder.version>
```

Comment:
please upgrade to the latest 35.0.1 version

Reply:
fixed